src/Sylius/Behat/Context/Api/Admin/ManagingOrdersContext.php

```diff
@@ -1267,6 +1267,15 @@ public function iShouldBeNotifiedThatTheOrdersPaymentCouldNotBeFinalizedDueToIns
         );
     }
 
+    /**
+     * @Then I should see that customer's IP address is :ipAddress
+     */
+    public function iShouldSeeCustomersIpAddress(
+        string $ipAddress,
+    ): void {
+        Assert::same($this->responseChecker->getValue($this->client->getLastResponse(), 'customerIp'), $ipAddress);
+    }
```

Comment:
```suggestion
    /**
     * @Then I should see this customer's IP address
     */
    public function iShouldSeeCustomersIpAddress(): void
    {
        Assert::notEmpty($this->responseChecker->getValue($this->client->getLastResponse(), 'customerIp'));
    }
```
I feel like checking the actual value is begging for problems; similar w/ UI